recipes/npcap/conandata.yml

```diff
@@ -0,0 +1,34 @@
+sources:
+  "1.01":
```

Comment:
Please, keep the newest version on the top. Also, only include versions that you REALLY need. If anyone need an older version, a new PR should be open, explaining the need.